popup.js

```diff
@@ -0,0 +1,359 @@
+                sign-ups required. Has been the industry's standard.
+            </p>
+            <ul class="tag">
+                <li class="tag-description-2">HTML</li>
+                <li class="tag-description-2">Bootstrap</li>
+                <li class="tag-description-2">Ruby</li>
+            </ul>
+        </div>
+        <button type="button" class="see-prj inter700 wide desktop-display-none">
+            See project
+        </button>
+    </div>
+
+
+    <div class="card-flex proArt card-type3 card6">
+        <div class="text-container card6-hidde">
+            <h3 class="card-title crete font-white desktop-display-none">
+                Profesional Art Printing Data
+            </h3>
+            <h3 class="card-title crete font-white display-none show-on-desktop">Website Portfolio</h3>
+            <p class="card-description inter400 font-white">
+                A daily selection of privately personalized reads; no accounts or
+                sign-ups required. Has been the industry's standard.
+            </p>
+            <ul class="tag">
+                <li class="tag-description-2">HTML</li>
+                <li class="tag-description-2">Bootstrap</li>
+                <li class="tag-description-2">Ruby</li>
+            </ul>
+        </div>
+        <button type="button" class="see-prj inter700 wide desktop-display-none">
+            See project
+        </button>
+    </div>
+
+</div>
+</section>`;
+
+const popupWindow = `    <div id="popup-container">
+<div id="popup-head">
+    <div id="popup-title-container">
+        <h2 id="popup-title" class="crete400"></h2>
+        <img id="popup-close-btn" src="./icons/closeIcon.svg" alt="close button">
+    </div>
+    <div id="popup-techs-container" class="poppins600">
+        <div id="tech1"></div>
+        <div id="tech2"></div>
+        <div id="tech3"></div>
+    </div>
+</div>
+<div id="popup-content">
+    <div id="popup-image-wrapper">
+        <img id="popup-image" src="" alt="portfolio image">
+    </div>
+    <div id="popup-text-btns-wrapper">
+        <p id="popup-description" class="inter400"></p>
+        <div id="popup-btns">
+            <a href="" id="popup-live-btn" class="see-prj inter700">
+                See live
+                <img src="./icons/liveIcon.svg" alt="live icon">
+            </a>
+            <a href="" id="popup-prj-btn" class="see-prj inter700">
+                See Source
+                <img src="./icons/ghIcon.svg" alt="github icon">
+            </a>
+        </div>
+    </div>
+</div>
+</div>`;
+
+const footer = document.querySelector('footer');
+const header = document.querySelector('header');
+
+window.addEventListener('load', () => {
+  const worksContainer = document.querySelector('#worksContainer');
+  worksContainer.innerHTML += worksContent;
+
+  const popupModal = document.querySelector('#popup-window');
+  popupModal.innerHTML += popupWindow;
+
```

Comment:
- [ ] As the previous reviewer mentioned, in the desktop view, your popup styles still don't look completely like the original design. I noticed that the Image was smaller than that of the original design and the popup card looks smaller compared to that of the original design so kindly work on that.:wink:
- [ ] Aside from that, the buttons are not visible so kindly work on that as well :100: 
- Your Design
![image](https://user-images.githubusercontent.com/102699958/232118671-94d25263-6a06-45b2-9faf-faff6d7f6784.png)

- The Orignal Design
![image](https://user-images.githubusercontent.com/102699958/232118714-1c3174ae-adf1-47e8-b30b-bf92f7207ee9.png)


- [ ] In the mobile view, just as the previous reviewer mentioned, the rest of the paragraph text doesn't show as well as the buttons. So kindly fix this as well and ensue that your design looks similar to thye original design :+1: 

- The Original Design
![image](https://user-images.githubusercontent.com/102699958/232119623-f91f161b-7914-4a23-a27d-c18aa08dbad4.png)

- Your Design
[Screencast from 04-14-2023 05:45:43 PM.webm](https://user-images.githubusercontent.com/102699958/232119542-8a9213b5-ebc8-4fc4-96ac-65a62cdd5377.webm)